scripts/reload_constants.sh

```diff
@@ -0,0 +1,13 @@
+cd ./scripts
+source ../.env
+
+PGPASSWORD=$POSTGRES_PASSWORD \
+psql -U $POSTGRES_USER \
+    -h localhost \
+    -p $DATABASE_PORT \
+    -v db_name=$POSTGRES_DB \
+    -d $POSTGRES_DB \
+    -f rebuild_constants.sql 
+
+cd ..
+python3 ./scripts/seed.py
```

Comment:
Use `chmod +x` to give this script executable permissions. Then use the shebang `#!/usr/bin/env python` at the top of the python script to invoke the python interpreter. This makes it much more flexible. Then you can get rid of the `python3` and invoke it just like a shell script.